### PR TITLE
Task/hmo common colors

### LINKED ISSUE #37: [Spec] Common colors
## Summary
The package should provide colors that can be used by the consumers. This will include the common `Theme` colors of a DIPS specific app, but also colors that can be used for status and other color use cases.

## API Changes

- Include a `Colors.xaml` file that can be referenced by our Consumers.
- Each `Color` should have a `x:Key`

e.g:
```xml
<ResourceDictionary xmlns="http://xamarin.com/schemas/2014/forms"
                    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
                    xmlns:d="http://xamarin.com/schemas/2014/forms/design"
                    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
                    mc:Ignorable="d"
                    x:Class="DIPS.Xamarin.UI.Styles.Colors">

    <Color x:Key="Theme.Primary">#047F89</Color>
```

## Intended Use Case
Consumers that use our library can easily use our colors by referencing the `.xaml` file in their `App.xaml`:
```xml
<ResourceDictionary.MergedDictionaries>
     <ResourceDictionary Source="pack://application:,,,/DIPS.Xamarin.UI;component;Styles/Colors.xaml"/>
</ResourceDictionary.MergedDictionaries>
```

And refer each color in line in a `.xaml` file:
```xml
Color={StaticResource ColorName}
```

### PR DESCRIPTION
**Pull request by haavamoa**
## Added / Fixed

- Added common Colors ([documentation](https://github.com/DIPSAS/DIPS.Xamarin.UI/wiki/Colors)), fixed #37 

## Todo List

- [X] I have tested on an Android device.
- [X] I have tested on an iOS device.
- [X] I have added unit tests - None added

>*Please add pictures / Gifs if possible*
>*Todo List can be checked by putting a `X` inside the brackets*
>*Remember to update our `wiki` after this PR is merged*

### CODE REVIEW

**Comment by haavamoa** [comment]
# Look and feel

colors_android

File: src/DIPS.Xamarin.UI/Resources/Colors/ColorPalette.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+using System.Collections.Generic;
+using System.Text;
+using Xamarin.Forms;
+
+namespace DIPS.Xamarin.UI.Resources.Colors
+{
+#pragma warning disable 1591
+    public static class ColorPalette
+    {
+        public static Color Dark = Color.FromHex("#000000");
```
Comment (Illedan) [inline src/DIPS.Xamarin.UI/Resources/Colors/ColorPalette.cs]: Disable comment inside and explain what this class of colors is meant for?

Comment (haavamoa) [reply src/DIPS.Xamarin.UI/Resources/Colors/ColorPalette.cs]: Move pragma disable down to property level and explain class? Sure thing, I will have to ask for an description from our designers

**Comment by haavamoa** [comment]
colors_android

**Pull request merged**